app/models/artist_public_data.rb

```diff
@@ -19,9 +19,32 @@ def path(context)
       if @artist.image.present?
         @artist.image.file.url
       end
+    else
+      "/artists/#{@artist.id}"
     end
   end
 
+  def birth_date
+    @artist.birth_date.to_s
+  end
+
+  def deceased_date
+    @artist.deceased_date.to_s
+  end
+
+  def bio
+    missing_bio = "Baltimore Public Art Commons is sourcing"          <<
+      " biographical information about artists from Wikipedia, but"   <<
+      " #{name(:show)} does not have an entry. If you would like to"  <<
+      " contribute, please create an entry."
```

Comment:
Can we change "please create an entry." to "please add the artist to Wikipedia."